### PR TITLE
Alumni name fix

### PR DESCRIPTION
**Pull request by 01jongmin**
Added check to see if the first name is an empty string and if so, just say Howdy!

### CODE REVIEW

File: PennMobile/Home/Controllers/HomeViewController.swift [hunk 1]
```diff
@@ -68,7 +68,13 @@ class HomeViewController: GenericViewController {
...
+                
+                if firstName != "" {
+                    self.displayTitle = "\(intros.random!), \(firstName)!"
+                } else {
+                    self.displayTitle = "Howdy!"
+                }
+                
                 titleCacheTimestamp = Date()
             } else {
                 self.displayTitle = "Home"
```
Comment (r3khaan) [inline PennMobile/Home/Controllers/HomeViewController.swift:80]: I think we should display the same message for users, who are not signed in, & alumni. Both should say "home" or both should give a customized greeting like "howdy." 

I like the latter option more, what do you think?

**Review by r3khaan — CHANGES_REQUESTED** [review]
Sweet and simple! Before merging in the PR though, I think we should make the experience consistent across users that are not logged in and alumni (see comment above)

**Comment by r3khaan** [comment]
@01jongmin

**Comment by 01jongmin** [comment]
Apologize for the delay. The code should now follow the later option.

**Review by r3khaan — CHANGES_REQUESTED** [review]
Very minor change! No need to re-request my review after, feel free to squash and merge yourself.

**Pull request merged**